### PR TITLE
merge upstream

### PR DESCRIPTION
**Pull request by wuchong**

### CODE REVIEW

File: containers/app.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  DISCOURSE_DEVELOPER_EMAILS: 'imjark@gmail.com'
+  ##
+  ## TODO: The domain name this Discourse instance will respond to
+  DISCOURSE_HOSTNAME: 'coderq.com'
+  ##
+  ## TODO: The mailserver this Discourse instance will use
+  DISCOURSE_SMTP_ADDRESS: smtp.mailgun.org         # (mandatory)
+  DISCOURSE_SMTP_PORT: 587                        # (optional)
+  DISCOURSE_SMTP_USER_NAME: postmaster@coderq.com      # (optional)
+  DISCOURSE_SMTP_PASSWORD: 5b7048fc83d52d066b4e34968a74d418               # (optional)
```
Comment (DeanMarkTaylor) [inline containers/app.yml]: @wuchong Is this supposed to be here?

**Comment by fundon** [comment]
I don't think so that should be merge into official repo.

**Comment by wuchong** [comment]
I'm sorry it's my mistake to pull a request . I want to delete it , but don't know how...

**Comment by riking** [comment]
Closing is enough, just stop commenting.

**Pull request closed without merging**